sparta/sparta/resources/Buffer.hpp

```diff
@@ -95,7 +95,7 @@ namespace sparta
          */
         struct DataPointer {
         private:
-            typename std::aligned_storage<sizeof(value_type), alignof(value_type)>::type object_memory_;
+            alignas(value_type) std::byte object_memory_[sizeof(value_type)];
```

Comment:
`std::aligned_storage` was deprecated in C++23: https://stackoverflow.com/questions/71828288/why-is-stdaligned-storage-to-be-deprecated-in-c23-and-what-to-use-instead